examples/eks-cluster-with-observability/README.md

```diff
@@ -0,0 +1,146 @@
+
+### Prerequisites
+
+- An existing Amazon Managed Grafana (AMG) Workspace.
+  - As of this writing (January 25, 2022), the AWS Terraform Provider does not support Amazon Managed Grafana, so it must be manually created beforehand. Instructions [here](https://docs.aws.amazon.com/grafana/latest/userguide/getting-started-with-AMG.html)
+
+#### Generate a Grafana API Key
+- Give the SSO user you set up in when creating the AMG Workspace admin access. 
+  - In the AWS Console, navigate to Amazon Grafana. In the left navigation bar, click __All workspaces__, then click __eks-ssp-observability__
+  - Under __Authentication__ within __AWS Single Sign-On (SSO)__, click __Configure users and user groups__
+  - Check the box next to the SSO user you created and click __Make admin__
+- Navigate back to the Grafana Dashboard. If you don't see the gear icon in the left navigation bar, log out and log back in.
+- Click on the gear icon, then click on the __API keys__ tab.
+- Click __Add API key__, fill in the _Key name_ field and select _Admin_ as the Role.
+- Copy your API key into `dev.tfvars` under `grafana_api_key` 
+
+### Deployment Steps
+
+- Clone this repository: `git clone https://github.com/aws-samples/aws-eks-accelerator-for-terraform.git`
+- Initialize a working directory:
+```
+cd examples/observability/eks-cluster-with-observability
+terraform init
+```
+- Fill-in the values for the variables in `dev.tfvars`
+  - The password for OpenSearch must be a minimum of eight characters with at least one uppercase, one lowercase, one digit, and one special character
+- Verify the resources created by this execution:
+```
+export AWS_REGION=<ENTER YOUR REGION>   # Select your own region
+terraform validate
+terraform plan -tf-vars=dev.tfvars
+```
+- Deploy resources with `terraform apply -tf-vars=dev.tfvars`
+- Add the cluster to your kubeconfig: `aws eks --region $AWS_REGION update-kubeconfig --name aws001-preprod-observability-eks`
+
+#### Map the FluentBit Role as a Backend Role to OpenSearch
+
+- Map the FluentBit Role as a Backend Role to OpenSearch:
+  - Replace "\<Your AWS Account ID\>", "\<Your Username\>", and "\<Your Password\>" with your own values.
+
+```
+export FLUENTBIT_ROLE="arn:aws:iam::<Your AWS Account ID>:role/aws001-preprod-observability-eks-aws-for-fluent-bit-sa-irsa"
+export ES_ENDPOINT=$(aws opensearch describe-domain --domain-name opensearch --output text --query "DomainStatus.Endpoint")
+export ES_DOMAIN_USER="<Your Username>"
+export ES_DOMAIN_PASSWORD='<Your Password>'
+
+curl -sS -u "${ES_DOMAIN_USER}:${ES_DOMAIN_PASSWORD}" \
+    -X PATCH \
+    https://${ES_ENDPOINT}/_opendistro/_security/api/rolesmapping/all_access?pretty \
+    -H 'Content-Type: application/json' \
+    -d'
+[
+  {
+    "op": "add", "path": "/backend_roles", "value": ["'${FLUENTBIT_ROLE}'"]
+  }
+]
+'
+```
+
+#### Deploy the Example Workload to Generate Logs and Metrics
+
+- Install ArgoCD https://argo-cd.readthedocs.io/en/stable/cli_installation/
```

Comment:
`enable_argocd = true`  should be part of this example. You can also bootstrap the cluster to auto deploy the sample workloads.

Reply:
`enable_argocd = true` is set in `main.tf` under `module "kubernetes-addons"`. Does it need to be set elsewhere? 

I'll update the the example so it auto deploys the sample workload.